application/__init__.py

```
@@ -25,12 +25,22 @@
 from flask import Flask, Blueprint, jsonify
 from werkzeug.exceptions import HTTPException
 from werkzeug.serving import WSGIRequestHandler
+from clp_logging.handlers import CLPFileHandler
```

Ruff (0.8.2)

`clp_logging.handlers.CLPFileHandler` imported but unused

(F401)